pegasus/test/test_form_routes.rb

```diff
@@ -30,30 +30,58 @@ class FormRoutesTest < SequelTestCase
     end
   end
 
-  DEFAULT_DATA = {
-    email_s: 'fake@example.com',
-    name_s: 'fake_name',
-    experience_s: 'university_student_or_researcher',
-    location_s: 'somewhere',
-    location_flexibility_ss: ['onsite'],
-    description_s: 'description',
-    allow_contact_b: '1',
-    age_18_plus_b: '1',
-    email_preference_opt_in_s: 'yes'
-  }.freeze
-
-  def test_volunteer_engineer_submission_2015
-    stubs(:dashboard_user).returns(nil)
-    Pegasus.stubs(:logger).returns(nil)
-    stubs(:request).returns(stub(ip: '1.2.3.4'))
-    row = insert_or_upsert_form('VolunteerEngineerSubmission2015', DEFAULT_DATA.dup)
-    @form = Form.find(id: row[:id])
-
-    @form.update(processed_data: {location_p: '37.774929,-122.419416'}.to_json)
-
-    assert_equal 0.8236209090344097,
+  describe 'volunteer_engineer_submission_2015' do
+    before do
+      # Delete all volunteer sign-ups in the test database before each test
+      ::PEGASUS_DB[:forms].where(kind: 'VolunteerEngineerSubmission2015').delete
+
+      Pegasus.stubs(:logger)
+      stubs(:dashboard_user)
+      stubs(:request).returns(stub(ip: '1.2.3.4'))
+    end
+
+    it 'returns local results' do
+      create_volunteer name: 'Local Person', location: '37.774929,-122.419416'
```

Comment:
This coordinate could be an Easter egg :)

Reply:
Market and Van Ness in San Francisco? I'm not sure what they're an easter egg for.